setup.py

```diff
@@ -32,7 +32,11 @@
     license='All Rights Reserved.',
     packages=find_packages(),
     # dependencies are named in requirements.pip
-    install_requires=[],
+    install_requires=[
```

Comment:
@hanpeter it's actually an anti-pattern. https://kruxdigital.jira.com/wiki/display/EN/How+to+set+up+transitive+dependencies+for+a+python+project